socorro/unittest/__init__.py

```diff
@@ -0,0 +1,15 @@
+from functools import total_ordering
+
+
+@total_ordering
+class EqualAnything(object):
+    def __eq__(self, other):
+        return True
+
+    def __lt__(self, other):
+        return True
+
+
+#: Sentinel that is equal to anything; simplifies assertions in cases where
+#: part of the value changes from test to test
+WHATEVER = EqualAnything()
```

Comment:
I wrote this for Fjord ages ago. It's super handy in tests where parts change test-to-test. Then you can do things like "assert this tuple is like this except for this index and this index which I don't care about" succinctly.